### DISCUSSION TITLE
Log file / location to show what chans are active in multi channel mode near real time

### DISCUSSION THREAD

**Discussion by KS4EC** [rtl-airband/RTLSDR-Airband#306] [Q&A] [answered]
I am building a semi headless feeder box for our local ham repeaters and am looking for a way to display channel status on a small display on the front of the box. Is there an easy way to tell what chans are active, like the waterfall display.

I am experimenting with an i2c LCD display as the front panel display. In the end there will be 34 chans across 4 SDR's so I was thinking of just putting the chan number with a "*" under it to display activity. I may have to reduce the number of chans to avoid buffer issues, but that's another issue.  I use Python for the LCD display but I am not real good at it "yet!"

Thanks in advance - Rob

**Comment by szpajder** [comment]
No such feature is currently available.

**Comment by charlie-foxtrot** [comment, marked as answer]
It would be a bit of a hack, but what about writing each channel to a file with `split_on_transmission = true` then use a python script to watch the output directory?

I *think* this would result in a file with a temporary file extension in the output directory whenever that channel is active, and that can be used to infer activity.

**Comment by KS4EC** [comment]
Interesting idea, as this is on a Raspberry pi and an SD card, do I need to worry about excessive writes to the SD card?
I'll dig into the write to file option and sort out watching the directory.  I am new to Python and linux so Google will be my friend.  
Thanks for the idea.
Rob

Sent from AT&T Yahoo Mail on Android 
 
  On Tue, Jan 18, 2022 at 7:23 PM, ***@***.***> wrote:   


It would be a bit of a hack, but what about writing each channel to a file with split_on_transmission = true then use a python script to watch the output directory?

I think this would result in a file with a temporary file extension in the output directory whenever that channel is active, and that can be used to infer activity.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by charlie-foxtrot** [comment]
If you mount a tmpfs and use that for your output directory the “files” will be written to RAM and not the SD card

**Comment by LoganSound** [comment]
I've been using inotifywait to watch folders this way, works well.
https://unix.stackexchange.com/questions/323901/how-to-use-inotifywait-to-watch-a-directory-for-creation-of-files-of-a-specific

**Comment by KS4EC** [comment]
That is cool, I'll have to try it as it might be a cleaner and faster way to do it.Thanks - Rob
    On Monday, September 12, 2022 at 03:02:46 PM EDT, HelicoptersofDC ***@***.***> wrote:  
 
 


I've been using inotifywait to watch folders this way, works well.
https://unix.stackexchange.com/questions/323901/how-to-use-inotifywait-to-watch-a-directory-for-creation-of-files-of-a-specific
